android/res/layout/fragment_routing.xml

```diff
@@ -23,7 +28,9 @@
     android:layout_height="1dp"
     android:background="?dividerHorizontal"/>
 
-  <include layout="@layout/start_button"
-      android:layout_width="match_parent"
-      android:layout_height="48dp"/>
+  <include
+    layout="@layout/start_button"
+    android:visibility="gone"
+    android:layout_width="match_parent"
+    android:layout_height="48dp"/>
```

Comment:
Hm...I just went throughout all layout.xml files in our project and there is no additional blank line before the closing tag of root container. And to be honest, I don't like the blank line in this place (because it seems redundant and doesn't make code better) I wouldn't change this style since we have it everywhere in our android project. Is it allowable for you?